cmd/testkms/main.go

```diff
@@ -13,20 +13,61 @@ import (
 	"strings"
 
 	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
+var datakeyopts = map[string]primitive.M{
+	"aws": bson.M{
+		"region": "us-east-1",
+		"key":    "arn:aws:kms:us-east-1:579766882180:key/89fcc2c4-08b0-4bd9-9f25-e30687b580d0",
+	},
+	"azure": bson.M{
+		"keyVaultEndpoint": "https://keyvault-drivers-2411.vault.azure.net/keys/",
+		"keyName":          "KEY-NAME",
+	},
+	"gcp": bson.M{
+		"projectId": "devprod-drivers",
+		"location":  "global",
+		"keyRing":   "key-ring-csfle",
+		"keyName":   "key-name-csfle",
+	},
+}
+
+var providers = func() []string {
```

Comment:
Optional: Consider moving this variable declaration into the `if !ok` block in `main` since that's the only place it's used.

E.g.
```go
func main() {
	// ...
	if !ok {
		// ...
		providers := make([]string, 0, len(datakeyopts))
		for p := range datakeyopts {
			providers = append(providers, p)
		}
		fmt.Println("- PROVIDER as a KMS provider, which supports:", strings.Join(providers, ", "))
		// ...
```